web/src/components/Firebase/Firebase.js

```diff
@@ -17,6 +20,75 @@ const firebaseClient = ((config) => {
   return firebase
 })(firebaseClientConfig)
 
+const FirebaseUserTools = () => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const { logIn, logOut, signUp, isAuthenticated } = useAuth()
+
+  return (
+    <div>
+      <h2>firebase</h2>
+      {isAuthenticated ? 'Authenticated' : 'Not Authenticated'}{' '}
```

Comment:
The extra space at the end shouldn't be needed with the `<h3>` following, right?
```suggestion
      {isAuthenticated ? 'Authenticated' : 'Not Authenticated'}
```

Reply:
Fixed.